xmipp

```diff
@@ -1209,6 +1266,26 @@ def getVersion(onlyNumbers=False):
 
     return cleanInfo.strip(' ')
 
+def getCudaVersion(nvcc):
+    log = []
+    runJob(nvcc + " --version", show_output=False, show_command=False, log=log)
+    # expected lst line: 'Cuda compilation tools, release 8.0, V8.0.61'
+    full_version = log[-1].strip().split(', ')[-1].lstrip('V')
+    tokens = full_version.split('.')
+    if len(tokens) < 2:
+        tokens.append('0')  # for version 5.0, only '5' is returned
```

Comment:
?

Reply:
Well, the comment it's wrong, but ensuring that `len(tokens) >=2` it's a good practice to avoid errors in the next line.